geps/overview.md

```diff
@@ -112,6 +117,14 @@ What is out of scope: see [text from KEP][kep-when-to-use]. Examples:
     key goal of GEPs is to show why we made a decision and which alternatives
     were considered. If separate docs are used, it's important that we can
     still see all relevant context and decisions in the final GEP.
+* Q: When should I mark a GEP as `Prototyping` as opposed to `Provisional`?
+  * A: The `Prototyping` status carries the same base meaning as `Provisional`
+    in that consensus is not complete between stakeholders and we're not ready
+    to move toward releasing content yet. You should use `Prototyping` to
+    indicate to your fellow community members that we're in a state of active
+    practical tests and experiments which are intended to help us learn and
+    iterate on the GEP. These can include distributing content, but not under
+        any release channel.
```

Comment:
Tiny nit: Looks like indentation got messed up here.

Reply:
Good catch, fixed in b6f08e228869dda0a5199c8a7a2e612534b64ec4 :+1: